src/Peering/Peering/help/Get-AzLegacyPeering.md

```diff
@@ -58,7 +58,7 @@ Gets the legacy peering for Seattle
 The credentials, account, tenant, and subscription used for communication with Azure.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
```

Comment:
@rileymckenna please regenerate the help content using the `-UseFullTypeName` switch parameter when running [`Update-MarkdownHelpModule`](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module) cmdlet